### PR TITLE
Panel

### PR DESCRIPTION
**Pull request by lwclwclwc60**

### CODE REVIEW

File: public/assets/panel/panel-statistics.js [hunk 1]
```diff
@@ -1,124 +1,163 @@
...
-                        display: true,
-                        text: xElement.innerHTML
-                    }
-                },
-                y: {
-                    title:{
-                        display: true,
-                        text: yElement.innerHTML
+        i = 1;
+        labels = Array(5).fill().map(()=>`${i}km around the MTR station`);
```
Comment (LoriIp) [inline public/assets/panel/panel-statistics.js:57]: labels = Array(5).fill().map((_, i)=>`${i+1}km around the MTR station`);

Comment (LoriIp) [inline public/assets/panel/panel-statistics.js:57]:
```suggestion
        labels = Array(5).fill().map((_, i)=>`${i+1}km around the MTR station`);
```

Comment (Wang-Ip) [inline public/assets/panel/panel-statistics.js]:
```suggestion
        labels = Array(5).fill().map((_, i)=>`${i+1}km around the MTR station`);
```
It always displays 1km,  make this change to fix

**Review by Wang-Ip — APPROVED** [review]
Overall looks good.

**Review by Wang-Ip — APPROVED** [review]

**Pull request merged**